common-test/src/main/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -445,6 +445,11 @@ private Statement withClusterOperator(Annotatable element,
                             String value = envVar.get("value").textValue();
                             ((ObjectNode) envVar).put("value", TestUtils.changeOrgAndTag(value, dockerOrg, dockerTag));
                         }
+                        // Set log level
+                        if (varName.matches("STRIMZI_LOG_LEVEL")) {
```

Comment:
You can use `equals()` here.

Reply:
yes, you are right, we don't have any regex and `equals()` will be a better decision